### PR TITLE
Hotfix - Saved List Bug

### PR DESCRIPTION
**Pull request by jaypeasee**
## Is this a fix or a feature?
Fix

## What does it fix?
Fix a toggling bug in the bucket list and visited list views/cards

## Where should the reviewer start?
Review ParkContainer and SavedCard

## How should this be tested?
Run the page, add a park to the bucket list, go to the bucket list, click to add it to visited, go to visited and check to make sure the [Remove from Bucket List] button is no longer there.

@saraho1123
@jaypeasee
@baileydunning

### CODE REVIEW

File: src/ParkContainer/ParkContainer.tsx [hunk 1]
```diff
@@ -39,26 +39,30 @@ const ParkContainer: React.FC<ParkContainerProps> = props => {
 
   const addToVisited = (parkCode: string) => {
     const addedPark = findChosenPark(parkCode)
-
+    addedPark.visited = true
```
Comment (saraho1123) [inline src/ParkContainer/ParkContainer.tsx:42]: Brilliant little fix here that fixed 2 things! Nice!

File: src/SavedCard/SavedCard.tsx [hunk 2]
```diff
@@ -9,9 +10,13 @@ const SavedCard: React.FC<SavedCardProps> = props => {
   const { name, parkCode, image, state, bucketList, visited } = props as LocalParkData
   const { addToVisited, deleteFromVisited, deleteFromBucketList } = props as AddRemoveFunctionality
 
+  useEffect(() => {
```
Comment (saraho1123) [inline src/SavedCard/SavedCard.tsx]: We should probably remove this useEffect, as we don't need it now.

**Review by saraho1123 — APPROVED** [review]
Nice teamwork to fix this bug!

File: src/ParkContainer/ParkContainer.tsx (hunk 1 above)
Comment (baileydunning) [reply src/ParkContainer/ParkContainer.tsx:42]: ❤️ two birds one stone

File: src/SavedParks/SavedParks.tsx [hunk 3]
```diff
@@ -36,8 +36,7 @@ const SavedParks: React.FC<LocalParkContainer & AddRemoveFunctionality> = props
         />
       })
       setVisitedListDisplay(visitedCards)
-
-  }, [visitedList])
+  }, [visitedList, bucketList])
```
Comment (baileydunning) [inline src/SavedParks/SavedParks.tsx:39]: Glad we caught how to re-render this condition & the source of this bug

**Review by baileydunning — APPROVED** [review]

**Review by saraho1123 — APPROVED** [review]

**Review by baileydunning — APPROVED** [review]

**Pull request merged**